README.md

```diff
@@ -1,3 +1,8 @@
+# End of Life Notice
+The MongoDB bson-haskell is now officially end-of-life (EOL). No further development, bugfixes, enhancements, documentation changes or maintenance will be provided by this project and pull requests will no longer be accepted.
```

Comment:
The reads a bit awkward. How about "The MongoDB bson-haskell library..."

Reply:
Updated, thank you Bernie!